### DISCUSSION TITLE
J44 collapse not working

### DISCUSSION THREAD

**Discussion by jpg13150** [templaza/astroid-framework#456] [Q&A] [answered]
I

The collapse component no longer works since the J4.4 migration

Framework Version: 3.0.4
Template Version: 3.0.3

Do you have a solution?

Thank you

<div class="container">
<h2>Example of a collapse element</h2>
<p>Click the button below to show or hide the text.</p>
<button type="button" class="btn btn-primary" data-bs-toggle="collapse" data-bs-target="#demo">Show/Hide</button>
<div id="demo" class="collapse">
<p>This is the collapse text. </p>
</div>
</div>

**Comment by sonvnn** [comment]
Please detail the issue. Which page do you get this issue?

**Comment by jpg13150** [comment]
I saw the problem on the page https://www.circuit-prive-en-thailande.fr/sejours-plage/sejour-a-phuket
when I clicked on a "tarif" button I had an area that became visible with the details of my rate
this worked perfectly before the J4.4 migration

I tried again on a test site where I only have joolma J 4.4 and Astroid to see if it didn't come from another extension
https://circuit-prive.joomla.com/index.php/collapse-astroid#collapseExample

This works under cassiopeia
https://circuit-prive.joomla.com/index.php/collapse-cassiopeia
If I select in the main menu plugin "collapsible dropdown"
![image](https://github.com/templaza/astroid-framework/assets/133236299/10a7f14d-7be1-42fb-8889-0620f0e4e492)

**Reply by sonvnn** [reply]
Thank you! I will check it and response you tomorrow. It's 22:00 here

**Reply by sonvnn** [reply, marked as answer]
This issue because Collapse Js is missing. Please go to admin -> template options -> Bootstrap Library -> select Collapse js. Then Astroid will load Collapse JS on your front-end

**Reply by sonvnn** [reply]
![screenshot-joomla templaza com-2023 11 17-11_06_48](https://github.com/templaza/astroid-framework/assets/3204103/c18581f5-8252-44e0-a16c-b17cb0a7881a)

**Comment by jpg13150** [comment]
Hello

It works perfectly

Thank you very much for your help
